Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/LogPoint_SIEM_Integration.py

```diff
@@ -0,0 +1,511 @@
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.Incidents.resolve',
+        outputs_key_field='',
+        outputs=msg
+    )
+
+
+def close_incidents_command(client, args):
+    incident_ids = args.get('incident_ids')
+    result = client.close_incidents(incident_ids)
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    msg = result.get('message')
+    markdown = "### " + msg
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.Incidents.close',
+        outputs_key_field='',
+        outputs=msg
+    )
+
+
+def reopen_incidents_command(client, args):
+    incident_ids = args.get('incident_ids')
+    result = client.reopen_incidents(incident_ids)
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    msg = result.get('message')
+    markdown = "### " + msg
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.Incidents.reopen',
+        outputs_key_field='',
+        outputs=msg
+    )
+
+
+def get_users_command(client):
+    result = client.get_users()
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    users = result.get('users')
+    table_header = []
+    if users and len(users) > 0:
+        table_header = list(users[0].keys())
+    markdown = tableToMarkdown('Incident Users', users, headers=table_header)
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix='LogPoint.Incidents.users',
+        outputs_key_field='id',
+        outputs=users
+    )
+
+
+def fetch_incidents(client, first_fetch):
+    """
+    This function retrieves new incidents every interval (default is 1 minute).
+    """
+    now = datetime.timestamp(datetime.utcnow())
+    last_run_object = demisto.getLastRun()
+    last_run = last_run_object and last_run_object.get('time', None)
+    if not last_run:
+        if first_fetch:
+            last_run = first_fetch
+        else:
+            last_run = datetime.timestamp(datetime.utcnow() - timedelta(days=1))
+    last_run = float(last_run)
+    result = client.get_incidents(last_run, now)
+    if not result.get('success'):
+        raise DemistoException(result['message'])
+    lp_incidents = result.get('incidents')
+    demisto.info("Executing LogPoint fetch_incidents between {} and {} Timestamp.".format(last_run, now))
+    incidents = []
+    # for alert in response:
+    for inc in lp_incidents:
+        detection_ts = inc['detection_timestamp']
+        dt = datetime.utcfromtimestamp(detection_ts)
+        occurred = dt.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+        incidents.append({
+            'name': inc.get('name', 'LgPoint - No name'),
+            'occurred': occurred,
+            'severity': get_demisto_severity(inc.get('risk_level')),
+            'rawJSON': json.dumps(inc)
+        })
+    demisto.setLastRun({'time': now})
```

Comment:
We made few extra changes apart from the suggested one in order to support `max_fetch` parameter. The `last_run` is set to `now` if number of incidents is less than `max_fetch`. Otherwise, `last_run` is set to the `detection_timestamp` of the next first incident to be fetched. We had to do that because in order to prevent repeated fetching.

Reply:
Nice!